lib/awful/screen.lua

```diff
@@ -1015,7 +1062,40 @@ object.properties(capi.screen, {
     auto_emit    = true,
 })
 
---@DOC_object_COMMON@
```

Comment:
?

Reply:
I'm confused about that. It keeps telling me there is nothing so get from upstream.

Reply:
I figured out what happened. I followed the build flags from the Archlinux repo to build it and I think there is something that changes the documentation comments. I developed the screen.lua live in /usr/local/share/awesome/lib and copied back to my home directory when it was working. This introduces the changes to the comments.

Now for some reason the patch is showing additions to client.lua and deletions from client.c. The file is correct, though. I can't figure out why that is.